packages/devextreme/testing/testcafe/helpers/accessibility/testAccessibility.ts

```diff
@@ -0,0 +1,91 @@
+import createWidget, { WidgetName } from '../createWidget';
+import { a11yCheck, A11yCheckOptions, ElementContext } from './accessibilityUtils';
+import { isMaterialBased } from '../themeUtils';
+
+export interface Options {
+  [key: string]: any[];
+}
+
+export interface Configuration {
+  component: WidgetName;
+  options?: Options;
+  a11yCheckConfig?: A11yCheckOptions;
+  selector?: ElementContext;
+  created?: (t: TestController, optionConfiguration?: Options) => Promise<void>;
+}
+
+const defaultSelector = '#container';
+const defaultOptions = {};
+const defaultCreated = async () => {};
+const defaultA11yCheckConfig = isMaterialBased() ? {
+  runOnly: 'color-contrast',
+} : {};
+
+const generateConfigurations = (
```

Comment:
let's split matrix test generation utils and accessebility test utils. It's just different functionality, we may use matrix generation for something else + i believe in grid we'll want to use accessebility checking without matrix or with complex matrixes where not all possible variants are ok

Reply:
Let's discuss it. I separated it (moved to helpers/generateOptionMatrix.ts). But:

`i believe in grid we'll want to use accessibility checking without matrix or ...`
I get it. In this case you don't need this functionality, you need to use a11yCheck from util.ts. But we want to be able to call just one function, passing possible options to it.